doc/en-us/dev/java/entity-ai/behavior_group.md

```diff
@@ -1,39 +1,39 @@
-# 第六章 行为组 - 组件的集合
+# Chapter 6 Behavior Groups - Collections of Components
 
 _**author: daoge_cmd**_
 
-**Waiting for translation, if you are interested in translation, welcome to contribute.**  
+**The translation made by AzaleeX contributed to the PowerNukkitX documentation**  
 
-## 1.0 所有组件的容器
+## 1.0 Containers for all components
 
-事实上，行为组```BehaviorGroup```并不只是行为的容器，其实际上包含了一类生物的全部逻辑组件，本章介绍的行为只是其组件的一部分。一个完整的行为组包含：
+In fact, the behavior group ```BehaviorGroup``` is not just a container for behaviors, but actually contains all the logical components of a class of creatures, and the behaviors presented in this chapter are only some of its components. A complete behavior group contains:
 
-- 行为，核心行为
-- 记忆存储
-- 传感器
-- 寻路器
-- 控制器
+- Behavior, Core Behavior
+- Memory Storage
+- Sensors
+- Pathfinder
+- Controller
 
-### 1.0.1 一个完整的AI
+### 1.0.1 A complete AI
 
-行为组包含了所有组件，即行为组组成了一个完整的AI
+The behavior group contains all components, i.e. the behavior group forms a complete AI
 
-生物在初始化时，会调用一次```requireBehaviorGroup()```方法，此方法返回的行为组实例将伴随此生物的整个生命周期
+The creature is initialized with one call to ```requireBehaviorGroup()``` method, which returns an instance of the behavior group that will accompany the creature for its entire lifetime
 
-通过覆写```EntityIntelligent```类的```requireBehaviorGroup()```方法并返回自己的行为组实例，生物就可以获得自己的AI
+By overriding the ```requireBehaviorGroup()``` method of the ```EntityIntelligent``` class and returning its own behavior group instance, the creature can obtain its own AI
 
-### 1.0.2 单例性
+### 1.0.2 Singularity
 
-对于每个生物，在其生命周期中只存在一个行为组实例
+For each organism, there is only one instance of the behavior group in its life cycle
```

Comment:
I think creature is better than organism here

Reply:
fix on this commit d476ae851733db3d3eca74a962c507167620b074